nifi-nar-bundles/nifi-dropbox-bundle/nifi-dropbox-services-api-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,100 @@
+nifi-dropbox-services-api-nar
+Copyright 2015-2022 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+  (ASLv2) The Dropbox Project
+     The following NOTICE information applies:
+       The Dropbox Project
+       Copyright 2013-2021 Dropbox Inc.
+
+  (ASLv2) The Netty Project
+    The following NOTICE information applies:
+      The Netty Project
+      Copyright 2011 The Netty Project
+
+  (ASLv2) Joda Time
+    The following NOTICE information applies:
+      This product includes software developed by
+      Joda.org (http://www.joda.org/).
+
+  (ASLv2) Apache Java Data Objects (JDO)
+    The following NOTICE information applies:
+      Apache Java Data Objects
+      Copyright 2005-2006 The Apache Software Foundation
+
+  (ASLv2) Apache HttpComponents
+    The following NOTICE information applies:
+      Apache HttpClient
+      Copyright 1999-2015 The Apache Software Foundation
+
+      Apache HttpCore
+      Copyright 2005-2015 The Apache Software Foundation
+
+  (ASLv2) Apache Commons Lang
+    The following NOTICE information applies:
+      Apache Commons Lang
+      Copyright 2001-2015 The Apache Software Foundation
+
+      This product includes software from the Spring Framework,
+      under the Apache License 2.0 (see: StringUtils.containsWhitespace())
+
+  (ASLv2) Apache Commons IO
+    The following NOTICE information applies:
+      Apache Commons IO
+      Copyright 2002-2016 The Apache Software Foundation
+
+  (ASLv2) Apache Commons FileUpload
+    The following NOTICE information applies:
+      Apache Commons IO
+      Copyright 2002-2014 The Apache Software Foundation
+
+  (ASLv2) Apache Commons Codec
+    The following NOTICE information applies:
+      Apache Commons Codec
+      Copyright 2002-2014 The Apache Software Foundation
```

Comment:
See other note about reviewing and removing unused library references.

Reply:
After removing the 3rd party lib from pom.xml, no external references remained.
The new NOTICE file contains the header only.